### DISCUSSION TITLE
MediaPicker CaptureVideoAsync iOS video resolution.

### DISCUSSION THREAD

**Discussion by ghost** [CommunityToolkit/Maui#850] [Q&A]
While using the capturevideoasync in media picker on iOS my video resolution is only 480x360.  In android I get higher quality.  Is there a configuration option or parameter that can be used for higher video capture resolution in iOS?

**Comment by CliffAgius** [comment]
Closed as per the Community Stand-up discussion here: https://youtu.be/N9wMcBP4jtg?t=2889
